app/components/CreateAccountSlide/index.js

```diff
@@ -87,106 +92,142 @@ const SeedsContainer = styled.div`
 `
 
 type Props = {
-  seed: string,
-  currentSlide: number,
-  nextAccountSlide: Function,
-  importAddress: Function,
-  generateNewMnemonic: Function
+  importAddress: Function
 };
 
-const setupSeedColumns = (seed) => {
-  const seedWords = seed.split(" ");
-  const seeds = [];
-  let seedColums = [];
+class CreateAccountSlide extends Component<Props> {
+  props:Props;
+  state = {
+    seed: '',
+    currentSlide: 0
+  };
 
-  seedWords.forEach((item, index) => {
-    seedColums.push(item);
-    if (index%5 === 4) {
-      seeds.push(seedColums);
-      seedColums = [];
-    }    
-  });
+  componentDidMount() {
+    this.updateMnemonic();
+  }
 
-  return seeds;
-}
+  nextAccountSlide = (currentSlide) =>  this.setState({ currentSlide });
+  updateMnemonic = () =>  this.setState({ seed: generateNewMnemonic() });
 
-const ShowSeedPhrase =  ({ seed, nextAccountSlide, generateNewMnemonic }) => {
-  const seeds = setupSeedColumns(seed);
-  return (
-    <Fragment>      
-      <div className='description'>
-        Write down your seed phrase on a piece of paper and keep it in a safe place. You will need your seed phrase to recover your account.
-      </div>
-      {seeds.length && 
+  setupSeedColumns = (seed) => {
+    const seedWords = seed.split(" ");
+    const seeds = [];
+    let seedColums = [];
+
+    seedWords.forEach((item, index) => {
+      seedColums.push(item);
+      if (index%5 === 4) {
```

Comment:
The `5` kind of feels like a magic number, might be good to make this a constant so you can just change it in one place later (I think this also impacts line 140?)